media/Jsontracer/colorList.js

```diff
@@ -0,0 +1,54 @@
+const colorList = [
```

Comment:
Can we keep this list with alphabetic order?

Reply:
Or does it mean to rearrange it in alphabetical order?
Since this is assigned from the color in front, it is difficult to do so because it has a visually good color in front.

Reply:
> The pretty colors are arranged in the front.

I didn't know this :D Thanks!

Reply:
When the number of nodes is small, only the color in front is shown, so I have a pretty color in front. In other words, it is meaningful in the current order.

![image](https://user-images.githubusercontent.com/42528189/135027469-ad5d3ef8-8a45-44c8-9823-ae44e4bb7278.png)